config/client_config/dc/system/config/templates/features/enroll_app/enroll_app.yml

```diff
@@ -878,10 +878,7 @@ registry:
         is_enabled: true
       - key: :prevent_concurrent_sessions
         item: :prevent_concurrent_sessions
-        is_enabled: <%= ENV['PREVENT_CONCURRENT_SESSIONS_IS_ENABLED'] || false %>
+        is_enabled: <%= ENV['PREVENT_CONCURRENT_SESSIONS_IS_ENABLED'] || true %>
```

Comment:
@TreyE We should default the behavior to false so that it behaves the way our current production behaves and also we are currently defaulting all our features to false(disabled state). We can use environment variables to disable or enable the feature.